concepts/fold/about.md

```diff
@@ -1,3 +1,57 @@
 # About
 
-TODO
+[`fold`][fold] is a [consuming iterator adaptor][consuming iterator adaptor] which applies a function to each element of the iteration, accumulating
+the result into a new value.
+
+Each iteration of `fold` takes two arguments. The first is an initial value for the accumulator. The second is a closure which itself takes
+two arguments: the accumulator and an item. The closure's return value is the subsequent value of the accumulator.
+Once the iterator drains, the final value of the accumulator is returned.
+
+`fold` can be used when it is desired to transform a collection into a single value. For example, if the sum of all even numbers is desired
+
+```rust
+pub fn main() {
+    let even_sum = (1..=10)
+        .fold(0, |acc, num| {
```

Comment:
```suggestion
        .fold(0, |mut acc, num| {
```

Just checked in the playground and I'd forgotten this `mut`.

Reply:
Ah! I checked `giant_grunts` but not this one. I should have seen it by eye, but my eyes were tired.

Reply:
https://github.com/exercism/rust/pull/1232/commits/1d8ad2ef2d0ae07104ec4d3a2d1df7606c12e990 doesn't actually add the `mut`; it just reformats.

Reply:
Nevermind, it doesn't need to be mutable in the new formatting. My bad.